waves/src/native/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+mod utils;
+use log::info;
+
+use crate::utils::set_panic_hook;
+use wasm_bindgen::prelude::*;
+
+// When the `wee_alloc` feature is enabled, use `wee_alloc` as the global
+// allocator.
+#[cfg(feature = "wee_alloc")]
+#[global_allocator]
+static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
+
+#[wasm_bindgen]
```

Comment:
Should this maybe be a `wasm_bindgen(start)`? https://rustwasm.github.io/wasm-bindgen/reference/attributes/on-rust-exports/start.html